siteConfig.yaml

```diff
@@ -30,9 +30,9 @@ stylesheets:
   - ./static/css/fixes.css
 
 #analytics:
-  # ga:
-  # note that GA doesn't work in the development environment
-  #  trackingId: UA-45997213-1
+# ga:
+# note that GA doesn't work in the development environment
+#  trackingId: UA-45997213-1
```

Comment:
I believe the indentation is meant to be meaningful here (nested YAML fields). Although it doesn't currently matter since it's commented out, can you back out these changes?